scripts/globals/crafting.lua

```diff
@@ -244,74 +446,79 @@ xi.crafting.unionRepresentativeTrade = function(player, npc, trade, csid, guildI
     end
 end
 
-xi.crafting.unionRepresentativeTrigger = function(player, guildId, csid, currency, keyitems)
+xi.crafting.guildPointNPConTrigger = function(player, csid, guildId)
```

Comment:
I'd prefer if we're calling this onTrigger to pass the two expected parameters (player, npc) to the function, and doing a lookup from a table in this script to determine what the guildId and other values are.

Reply:
(Also, I can't help but read this as NP-Con)

Reply:
I can change the parameters.
The consensus back then was that "Union Representative" was not really known/clear.
Would you preffer `guildPointOnTrigger`?

Reply:
I can get on board with this